spec/lib/field-types.js

```diff
@@ -1,6 +1,22 @@
 const Overwrite = require('@tangle/overwrite')
+const isBase64 = require('is-canonical-base64')()
```

Comment:
:fire: I think json schema patterns need to be strings, but I could be wrong, and this is a regexp

in past I've done:

```js
pattern: isBase64.toString().slice(1, -1) // the slice cuts off the start/end '/'
```

Reply:
I suggest wring a simple test for this fieldType and seeing what that says

Reply:
I think you're right, changed the value to a random one and it didnt fail, writing test now :)

Reply:
I think you're right, changed the value to a random one and it didnt fail, writing test now :)